app/src/main/java/fr/free/nrw/commons/auth/SessionManager.java

```diff
@@ -146,7 +141,6 @@ public Completable logout() {
         return Completable.fromObservable(Observable.fromArray(allAccounts)
                 .map(a -> accountManager.removeAccount(a, null, null).getResult()))
                 .doOnComplete(() -> {
-                    mediaWikiApi.logout();
```

Comment:
Just to confirm, if we remove the mediaWikiApi.logout(), how is the actual logout handled?

Reply:
@ashishkumar468 Can we go ahead and merge this to backend-overhaul and take up logout as a separate task?

Reply:
Sure